packages/react/tests/LocalNavigation.test.tsx

```diff
@@ -27,24 +27,27 @@ describe('react: LocalNavigation', () => {
     }
   });
 
-  it('should handle click events', () => {
-    const onClick = jest.fn();
+  // we should not be testing for onClick explicitly as react-aria suggests that we need to use onPress
+  // which will internally handle all the interactions properly
```

Comment:
[Comment - Blocking] I don't understand. We have the expectation that the navigation would have button press of some sort. So in the case of what you're saying we would use onPress. So the fix potentially would be to use the onPress method and update the test. I guess my questions is why are we removing the test?

Reply:
@smartinelli-p44  onPress is already tested in another test ( https://github.com/project44/manifest/blob/main/packages/react/tests/LocalNavigation.test.tsx#L49 ). So i commented out this one, and kept it in code so this act as an context that why we are not testing onClick, which we generally do.